E/Extrae/build_tarballs.jl

```diff
@@ -51,39 +56,53 @@ autoreconf -fvi
     --disable-openmp \
     --disable-nanos \
     --disable-smpss \
-    --without-mpi \
-    ${ENABLE_CUDA:+--with-cuda=$prefix/cuda} \
+    ${FLAGS_MPI:---without-mpi} \
+    ${FLAGS_CUDA:---without-cuda} \
     --with-binutils=$prefix \
     --with-unwind=$prefix \
-    --with-xml-prefix=$prefix \
+    --with-xml=$prefix \
     --with-papi=$prefix
 
 make -j${nproc}
 make install
 """
 
-non_cuda_platforms = [
+platforms = [
     Platform("x86_64", "linux"; libc="glibc"),
     Platform("aarch64", "linux"; libc="glibc"),
     Platform("powerpc64le", "linux"; libc="glibc"),
 ]
-non_cuda_platforms = expand_cxxstring_abis(non_cuda_platforms)
+platforms = expand_cxxstring_abis(platforms)
 # (Almost) Same as PAPI
 cuda_platforms = CUDA.supported_platforms(; max_version=v"12.2")
```

Comment:
I think we can do like PAPI: https://github.com/JuliaPackaging/Yggdrasil/blob/e48091185c15cd2807bca8ad3966910ff4aadb8e/P/PAPI/build_tarballs.jl#L94